inngest-spring-boot-demo/src/main/java/com/inngest/springbootdemo/InngestSingleton.java

```diff
@@ -0,0 +1,73 @@
+package com.inngest.springbootdemo;
+
+import com.inngest.*;
+import kotlin.jvm.functions.Function2;
+
+import java.time.Duration;
+import java.util.HashMap;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+
+class Result {
+    @JsonProperty("sum")
+    private final int sum;
+
+    public Result(@JsonProperty("sum") int sum) {
+        this.sum = sum;
+    }
+
+    public int getSum() {
+        return sum;
+    }
+}
+
+// NOTE: We probably don't need this singleton anymore
```

Comment:
The current SDK standards are to no expose any of the handlers to the users, and they can either,
- mount our route onto their HTTP controller
- directly create the HTTP endpoint for them

I'd imagine Java will be the former, so the singleton shouldn't exist and only the controller should be there.